docs/privacy-policy.md

```diff
@@ -0,0 +1,172 @@
+Privacy Policy
+==============
+
+**DRAFT**
+
+Your personal privacy is an important factor that the Sempervirens project and Numfocus 
+(our legal entity) considers in the development of each of our products and services. 
+We are committed to NOT storing ANY personal information through Sempervirens other 
+than IP addresses. We are committed to being transparent and open. This Privacy Policy 
+explains generically how we receive information from you, and what we do with that 
+information once we have it.
+
+What do we mean by "personal information?"
+------------------------------------------
+
+The phrase "personal information" means information which identifies you, such as your 
+name, email address, twitter account names, or other user names. Any information that 
+falls outside of this is "non-personal information."
+
+Combining non-personal information with personal information we consider to be 
+"personal information." Such combinations will not be publically shared. Removing all 
+personal information creates a data set that is deemed "non-personal information" 
+and may be collected.
+
+In some jurisdictions around the world, IP addresses and other similar pieces of
+information are considered personal information, though they are not "uniquely 
+identifying personal information." In an effort apply as broadly as possible, 
+Sempervirens considers IP addresses and other "non-uniquely identifying personal 
+information" to be personal information. 
+
+IP addresses are the only form of personal information that Sempervirens will collect
+about you. IP addresses will be stored as part of a standard web log and non-public 
+dataset. For example, IP addresses may be used internally by Sempervirens for
+server operations, debugging, and other server-side operations.
+IP addresses will never be shared or displayed publicly.
+
+Aggregating and anonymizing IP addresses creates a data set that is deemed 
+non-personal information.
+
+
+How do we learn information about you?
```

Comment:
Eventually we are going to have to give a much more complete and detailed description of the information that we will enable packages to collect.